src/libsync/propagatedownload.cpp

```diff
@@ -432,24 +432,23 @@ void PropagateDownloadFileQNAM::setDeleteExistingFolder(bool enabled)
     _deleteExisting = enabled;
 }
 
-const char owncloudCustomSoftErrorStringC[] = "owncloud-custom-soft-error-string";
 void PropagateDownloadFileQNAM::slotGetFinished()
 {
     _propagator->_activeJobList.removeOne(this);
 
     GETFileJob *job = qobject_cast<GETFileJob *>(sender());
     Q_ASSERT(job);
+    QNetworkReply::NetworkError err = job->replyError();
 
-    qDebug() << Q_FUNC_INFO << job->reply()->request().url() << "FINISHED WITH STATUS"
-             << job->reply()->error()
-             << (job->reply()->error() == QNetworkReply::NoError ? QLatin1String("") : job->reply()->errorString())
-             << _item->_httpErrorCode
+    qDebug() << Q_FUNC_INFO << job->replyUrl() << "FINISHED WITH STATUS"
+             << err
+             << (err == QNetworkReply::NoError ? QLatin1String("") : job->replyErrorString())
+             << _item->_httpErrorCode // change to job->replyHttpStatusCode() ?
```

Comment:
Is `_item->_httpErrorCode` the same that the HTTP Status code of the reply object in this context ?
If yes, we can standardise and use `job->replyHttpStatusCode()`